guides/lagom-dcos-marathon-deploy-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,392 @@
+cassandra                192.168.65.121   root     R    cassandra.f1595d39-c313-11e7-a4d4-70b3d5800001                ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S4  
+chirpservice.chirper     192.168.65.121   root     R    chirper_chirpservice.9b01861d-c314-11e7-a4d4-70b3d5800001     ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S4  
+friendservice.chirper    192.168.65.111   root     R    chirper_friendservice.9b015f0c-c314-11e7-a4d4-70b3d5800001    ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S5  
+friendservice.chirper    192.168.65.111   root     R    chirper_friendservice.a868f62f-c315-11e7-a4d4-70b3d5800001    ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S5  
+friendservice.chirper    192.168.65.121   root     R    chirper_friendservice.a8691d40-c315-11e7-a4d4-70b3d5800001    ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S4  
+front-end.chirper        192.168.65.111   root     R    chirper_front-end.9b0110eb-c314-11e7-a4d4-70b3d5800001        ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S5  
+marathon-lb              192.168.65.60    root     R    marathon-lb.48056504-c312-11e7-97a5-70b3d5800001              ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S7  
+node-0-server            192.168.65.121  nobody    R    node-0-server__b1a19fad-56be-4650-81ce-9ff2f6c25282           ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S4  
+node-1-server            192.168.65.131  nobody    R    node-1-server__f1820f91-70e6-4780-b1f2-5b47faa8cb3a           ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S6  
+node-2-server            192.168.65.111  nobody    R    node-2-server__93c62119-8b85-41bd-985e-6d7141ca4be9           ac2d39b3-db25-4a9b-a3ef-cc3b97fb5bd4-S5  
+```
+
+<pre class="code-bash prettyprint prettyprinted">
+dcos task log chirper_friendservice.a868f62f-c315-11e7-a4d4-70b3d5800001
+</pre>
+
+```
+2017-11-06T17:13:18.027Z [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=friendservice-akka.actor.default-dispatcher-2, akkaTimestamp=17:13:18.027UTC, akkaSource=akka.tcp://friendservice@192.168.65.111:14858/user/readSideGlobalPrepare-FriendEventProcessor-singleton, sourceActorSystem=friendservice] - ClusterSingletonManager state change [Start -> Younger]
+2017-11-06T17:13:18.126Z [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=friendservice-akka.actor.default-dispatcher-18, akkaTimestamp=17:13:18.126UTC, akkaSource=akka.tcp://friendservice@192.168.65.111:14858/user/cassandraOffsetStorePrepare-singleton, sourceActorSystem=friendservice] - ClusterSingletonManager state change [Start -> Younger]
+2017-11-06T17:13:18.129Z [info] akka.cluster.singleton.ClusterSingletonManager [sourceThread=friendservice-akka.actor.default-dispatcher-3, akkaTimestamp=17:13:18.128UTC, akkaSource=akka.tcp://friendservice@192.168.65.111:14858/system/sharding/FriendEntityCoordinator, sourceActorSystem=friendservice] - ClusterSingletonManager state change [Start -> Younger]
+```
+
+## Conclusion
+
+DC/OS provides many features that complement running a Lagom system in production. By leveraging 
```

Comment:
same comment as before, instead of "complement" maybe "many features that a Lagom system needs to run in production?

Reply:
You could end the sentence with a colon and have the next few sentences be bullets.